build.gradle

```
@@ -29,7 +29,11 @@ task set_conf_options(type:Exec){
 
 task copy_rtd_logo(type: Copy) {
     dependsOn set_conf_options
-    from file("${rootProject.rootDir}/repo_rtd_temp/logo.png")
+    if("${projectDir.parent.contains("RTD-UM")}"){
```

```suggestion
    if("${projectDir.parent.contains("RTD_UM")}" || "${projectDir.parent.contains("RTD-UM")}"){
```
Existen ciertos manuales cuyo nombre es RTD_UM en lugar de RTD-UM. Lo más correcto sería corregir estos manuales para que su codificación sea la correcta (no nos tocaría a nosotros corregirlos sino a los responsables de dichos manuales), pero, mientras esta corrección sucede se puede controlar este problema añadiendo este caso en la condición del if